src/pages/MyEarnings/MultipleChainSelectV2/index.tsx

```diff
@@ -78,20 +88,40 @@ const MultipleChainSelect: React.FC<MultipleChainSelectProps> = ({ className, st
     }
   }
 
+  useEffect(() => {
+    setLocalSelectedChains(selectedChains)
+    // eslint-disable-next-line
```

Comment:
:viet-nv:

Reply:
clone from https://github.com/KyberNetwork/kyberswap-interface/blob/107a4ba83d99b0ff42a2125d7600d25cc5ca38d6/src/pages/MyEarnings/MultipleChainSelect/PopoverBody.tsx#L116

will combine MulileChainSelect vs MulileChainSelectV2  later , will check it later.